doc/man1/openssl-verification-options.pod

```diff
@@ -653,9 +678,37 @@ set.
 
 =item B<CRL Signing CA>
 
-The normal CA tests apply. Except in this case the basicConstraints extension
+The normal CA checks apply. Except in this case the basicConstraints extension
 must be present.
 
+
+=item B<OCSP helper>
+
+For the target certificate special checks apply; see L<OCSP_basic_verify(3)>.
+For all other certificates the normal CA checks apply.
+
+=item B<Timestamp signing>
+
+For the target certificate the following checks apply.
+If the keyUsage extension is present, it must have the digitalSignature
+or nonRepudiation bit set, while all other bits must not be set.
+If the extendedKeyUsage extension is present, it must be marked critical
+and include the B<timeStamping> OID and must not include any other EKUs.
+
+For all other certificates the normal CA checks apply.
+
+=item B<Code signing>
+
+For the target certificate the following checks apply.
```

Comment:
```suggestion
For the end-entity certificate the following checks apply.
```

Reply:
No, this can also is the case for CA certs if they happen to be the target cert.